ctk/idp/src/main/resources/saml.xml

```diff
@@ -0,0 +1,118 @@
+<saml2p:Response xmlns:saml2p="urn:oasis:names:tc:SAML:2.0:protocol" xmlns:xsd="http://www.w3.org/2001/XMLSchema" Destination="https://samlhost:8993/services/saml/sso/post" ID="_fc259ded-3d6e-402a-a74a-365c23a1782d" InResponseTo="a1chfeh0234hbifc1jjd3cb40ji0d49" IssueInstant="2018-04-10T17:03:50.642Z" Version="2.0">
```

Comment:
❓ Did you mean to add this file? I'm assuming you used it for testing, but we probably don't need it in the project.

Reply:
Nope, it was just for testing. I'll remove it